python/lsst/ap/association/diaPipe.py

```diff
@@ -244,6 +244,12 @@ class DiaPipelineConfig(pipeBase.PipelineTaskConfig,
         target=DiaObjectCalculationTask,
         doc="Task to compute summary statistics for DiaObjects.",
     )
+    doLoadForcedSources = pexConfig.Field(
+        dtype=bool,
+        default=True,
+        doc="Load forced DiaSource history from the APDB?"
+            "This should only be turned off for debugging purposes.",
+    )
```

Comment:
I feel like this docstring should say something about how this was added to let us turn off forced sources for performance reasons? Maybe something like this?

```suggestion
        doc="Load forced DiaSource history from the APDB?"
            " This should only be turned off for debugging purposes."
            " Added to allow disabling forced sources for performance reasons during the ops rehearsal.",
    )
```